watchman/checks.py

```diff
@@ -66,13 +76,35 @@ def _check_email():
     return response
 
 
-def caches_status(request):
+def _check_storage():
+    try:
+        filename = 'django-watchman-{}.txt'.format(uuid.uuid4())
+        content = 'django-watchman test file'
+        path = default_storage.save(filename, ContentFile(content))
+        default_storage.size(path)
+        contents = default_storage.open(path).read()
```

Comment:
I was comparing `content` vs. `contents` (maybe not the best names :smile:), but I was getting some strange errors on Python 3 in my tests. I pulled it out for now, and opened a separate issue for it: #26